ansible/ansible.cfg

```diff
@@ -0,0 +1,17 @@
+[ssh_connection]
+pipelining              = True
+#ssh_args               = -o ControlMaster      = auto -o ControlPersist = 30m -o ConnectionAttempts = 100 -o UserKnownHostsFile = /dev/null
```

Comment:
Is this on purpose commented out?

Reply:
A leftover, will take the commented stuff out.